app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -310,6 +313,35 @@ private void downloadBook(String url, int notificationID, LibraryNetworkEntity.B
             notification.get(notificationID).setContentText(getString(R.string.zim_file_downloaded));
             final Intent target = new Intent(this, KiwixMobileActivity.class);
             target.putExtra(EXTRA_ZIM_FILE, KIWIX_ROOT + StorageUtils.getFileNameFromUrl(book.getUrl()));
+            //Remove the extra ".part" from files
```

Comment:
Removes the additional `.part ` from the downloaded chunks once book has finished downloading.